iset.mm

```diff
@@ -29795,6 +29795,12 @@ intersection or the difference (that is, this theorem would be equality
       FPQGRSTCBUHDLUGUHCDUHFUINCDOUHUIFAEFUAQHRSUCUGBAUHUBCAUDUEUF $.
   $}
 
+  $( Identical true and false arguments in the conditional operator.
+     (Contributed by NM, 18-Apr-2005.) $)
+  ifiddc $p |- ( DECID ph -> if ( ph , A , A ) = A ) $=
```

Comment:
I was curious and had a look at df-if. It looks like we have
```
|- ( A e. if ( ph , B , C ) -> DECID ph )
```
?

Reply:
That's cute. Would indeed seem to follow from df-if and emphasizes even more the point that `if` doesn't do much unless `ph` is decidable.